src/index.js

```diff
@@ -263,6 +272,12 @@ const onMapLoad = async () => {
 // load map
 map.on('load', onMapLoad)
 
+// add toggle-button handler
+let button = document.getElementById('toggle-button')
```

Comment:
Nitpick/I’m not actually sure, could this be a const?

Reply:
Also we could just name it toggle button to avoid future confusion with other buttons if they get added

Reply:
Done!